.travis.yml

```diff
@@ -91,6 +91,20 @@ jobs:
           on:
             tags: true
             repo: dita-ot/dita-ot
+    # Deployment
+    - state: deploy
+      name: "Deploy Docker"
+      install: skip
+      script: skip
+      deploy:
+        - provider: script
+          script: |-
+            docker login docker.pkg.github.com -u $GITHUB_USER -p $GITHUB_PACKAGE_REGISTRY
+            TAG=${TRAVIS_TAG} docker build -t docker.pkg.github.com/dita-ot/dita-ot/dita-ot:${TRAVIS_TAG} --build-arg VERSION=${TRAVIS_TAG} .
+            TAG=${TRAVIS_TAG} docker push docker.pkg.github.com/dita-ot/dita-ot/dita-ot:${TRAVIS_TAG}
```

Comment:
Since this deployment doesn't build the package, only downloads it from GitHub, this can be ran locally to build image for older versions of DITA-OT.